TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -235,7 +235,76 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
```

Comment:
Parsers for pingperf baremetal test and techempower seem to be missing here as well. Can we please add them along with these?